benchmarks/coin_store.py

```diff
@@ -26,13 +26,13 @@ def make_coin() -> Coin:
     return Coin(rand_hash(), rand_hash(), uint64(1))
 
 
-def make_coins(num: int) -> Tuple[List[Coin], List[bytes32]]:
+def make_coins(num: int) -> Tuple[List[Coin], List[CoinID]]:
     additions: List[Coin] = []
-    hashes: List[bytes32] = []
+    hashes: List[CoinID] = []
     for i in range(num):
         c = make_coin()
         additions.append(c)
-        hashes.append(c.name())
+        hashes.append(CoinID(c.name()))
```

Comment:
The title says `TransactionRecordID`, so I assume all these `CoinID` changes are mistakes, is that right?